command/server.go

```diff
@@ -50,6 +50,8 @@ import (
 var _ cli.Command = (*ServerCommand)(nil)
 var _ cli.CommandAutocomplete = (*ServerCommand)(nil)
 
+const migrationLock = "sys/migration"
```

Comment:
I think this makes more sense as `core/migration`, sys is usually reserved for system backend things.

Reply:
Changed.